chatbot_ner/config.py

```diff
@@ -39,6 +40,35 @@
 ner_logger.addHandler(handler)
 ner_logger.addHandler(handler_stdout)
 
+# HAPTIK Environment and CAS name
+CLIENT_APPLICATIONS_SETUP_NAME = os.environ.get('CLIENT_APPLICATIONS_SETUP_NAME')
+HAPTIK_ENV = os.environ.get('HAPTIK_ENV')
+
+# Support for Sentry DSN
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
+SENTRY_ENABLED = os.environ.get('SENTRY_ENABLED')
+SENTRY_ENABLED = True if SENTRY_ENABLED == 'True' and 'test' not in sys.argv else False
+SENTRY_ALLOWED_LOGGERS = os.environ.get('SENTRY_ALLOWED_LOGGERS')
+SENTRY_ALLOWED_LOGGERS = [sentry_allowed_logger.strip() for sentry_allowed_logger
+                          in SENTRY_ALLOWED_LOGGERS.split(',')] if SENTRY_ALLOWED_LOGGERS else []
```

Comment:
These seem to be unused.

Also, I think we should be maintaining `SENTRY_IGNORED_LOGGERS` instead of `SENTRY_ALLOWED_LOGGERS` because sentry sdk also gives the option to ignore rather than whitelist